app/.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_FORM_ACTION_URL=https://formcarry.com/s/OlAl-K2DqOh
```

Comment:
this should not be public

please check https://vercel.com/docs/projects/environment-variables

Reply:
okay, understood! done
the env variables were included in vercel section

Reply:
then please remove it from the code